src/timesync/timesyncd-server.c

```diff
@@ -60,6 +60,14 @@ ServerAddress* server_address_free(ServerAddress *a) {
         return mfree(a);
 }
 
+static void enable_ntp_server_defer_event(Manager *m, ServerType type) {
+        int r = sd_event_source_set_enabled(m->deferred_ntp_server_event_source, SD_EVENT_ONESHOT);
```

Comment:
`r` gets special treatment for us across the codebase, it's always declared up top, on its own, as the primary variable that error returns are written to.

hence change to:

```c
int r;

r = …
```